Library/Homebrew/test/language/node_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "language/node"
+
+describe Language::Node do
+  specify "#npm_cache_config" do
+    shutup do
+      ret_val = described_class.npm_cache_config
+      expect(ret_val).to eq("cache=#{HOMEBREW_CACHE}/npm_cache\n")
+    end
+  end
+
+  describe "#pack_for_installation" do
```

Comment:
This method is called by `#setup_npm_environment` so I don't think a separate test that's longer than the actual method is warranted.